Setup/UpgradeData.php

```diff
@@ -169,4 +182,42 @@ public function updateSchemaVersion800(ModuleDataSetupInterface $setup)
             ['path = ?' => 'payment/adyen_abstract/split_payments_refund_strategy']
         );
     }
+
+    /**
+     * If Store alternative payment methods is on, turn the config off, since it was previously NOT operational.
+     * This will ensure that if this config is turned back on, the Token type will also be saved.
+     *
+     * @param ModuleDataSetupInterface $setup
+     */
+    public function updateSchemaVersion821(ModuleDataSetupInterface $setup)
+    {
+        $configDataTable = $setup->getTable('core_config_data');
+        $pathStoreAlternativePaymentMethod = 'payment/adyen_hpp_vault/active';
+        $connection = $setup->getConnection();
+
+        $select = $connection->select()
+            ->from($configDataTable)
+            ->where(
+                'path = ?',
+                'payment/adyen_hpp_vault/active'
```

Comment:
```suggestion
                $pathStoreAlternativePaymentMethod
```

Reply:
Updated in [5bee1ec](https://github.com/Adyen/adyen-magento2/pull/1389/commits/5bee1eca1388d16f6858c89bcf668df474611f35)